Todo.js

```diff
@@ -0,0 +1,43 @@
+import { acceptedTodo, deleteTodo } from './main'
```

Comment:
there is also an additional rule called import order.
You need to keep your imports in the special order:
```
1. imports from node_modules
2. other imports
```